crates/vim/src/state.rs

```diff
@@ -933,7 +937,8 @@ impl Operator {
             Operator::Object { around: false } => "i",
             Operator::Object { around: true } => "a",
             Operator::Change => "c",
-            Operator::Delete => "d",
+            Operator::Delete { inclusive: false } => "d",
+            Operator::Delete { inclusive: true } => "v",
```

Comment:
I'm lower context on this - it seems fine? My only question would be, could future edits to `vim.json` spuriously allow a user to directly access the `PushDeleteInclusive` operator, rather than traveling first through the `PushDelete` operator? And if so, is that bad?

Reply:
If i'm understanding the question correctly then yeah I think so. 
Someone would just need to move the [pushdeleteinclusive mapping](https://github.com/zed-industries/zed/pull/27991/files/8cc430f023f18a2e68e6cc8404faee8fc72cf8b4#diff-7b7dabbece7a54e041c4b6ab8f69afbe994d301dbded14921a31598d7f6de287R456) out from under the [d operator context](https://github.com/zed-industries/zed/pull/27991/files/8cc430f023f18a2e68e6cc8404faee8fc72cf8b4#diff-7b7dabbece7a54e041c4b6ab8f69afbe994d301dbded14921a31598d7f6de287L452).
But that is true for anything in that section so I don't think it is concern.